database/insertData/04_users.js

```diff
@@ -99,7 +99,35 @@ exports.queries = [
         user: { email: "reviewer2@sussol.net"
           passwordHash: "$2a$10$pzhH6GcC7rw38AencBcbCuDaN6ANGZnVnE3ViCa5veOeTelbkkkv2"
           username: "testReviewer2",
-          firstName: "Mrs", lastName: "Reviewer 2" }
+          firstName: "Reviewer", lastName: "2" }
+      }
+    ) {
+      user {
+        username
+      }
+    }
+  }`,  
+  `mutation {
+    createUser(
+      input: {
+        user: { email: "consolidator1@sussol.net"
+          passwordHash: "$2a$10$pzhH6GcC7rw38AencBcbCuDaN6ANGZnVnE3ViCa5veOeTelbkkkv2"
```

Comment:
Strictly speaking they should all have unique hashes. Not that it really matters for testing purposes, but if you can be bothered: https://www.browserling.com/tools/bcrypt

Reply:
Ah, nah. For now we want them to all have the same, so we can easily change from one to another